### PR TITLE
Updated for v4

### PR DESCRIPTION
**Pull request by dogversioning**
- touched manifest
- added explicit prefix in counts.py

### CODE REVIEW

File: cumulus_library_suicidality_icd10/manifest.toml [hunk 1]
```diff
@@ -23,33 +13,13 @@ file_names = [
     "table_study_period.sql",
     "table_dx.sql",
     "table_prevalence.sql",
-    "chartreview_case.sql",
-    "version.sql"
-]
-
-[counts_builder_config]
-file_names = [
+    #"chartreview_case.sql",
```
Comment (mikix) [inline cumulus_library_suicidality_icd10/manifest.toml:16]: What happened with this file?

File: cumulus_library_suicidality_icd10/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = "1.0.0"
+__version__ = "1.1.0"
```
Comment (mikix) [inline cumulus_library_suicidality_icd10/__init__.py:1]: Minor bump is fine, but in my head a major bump on the runner might be worth a major bump here too? Don't actually care, just noting it.

**Review by mikix — APPROVED** [review]

File: cumulus_library_suicidality_icd10/manifest.toml (hunk 1 above)
Comment (dogversioning) [reply cumulus_library_suicidality_icd10/manifest.toml:16]: @comorbidity pulled it - https://github.com/smart-on-fhir/cumulus-library-suicidality-icd10/commit/e2189153160f10e89edc0f72808a113a61bea975

PRs are now required for commits to main so this doesn't happen again.

File: cumulus_library_suicidality_icd10/__init__.py (hunk 2 above)
Comment (dogversioning) [reply cumulus_library_suicidality_icd10/__init__.py:1]: fair point, i'll update

**Pull request merged**